sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -43,12 +43,10 @@ typedef struct
 {
   /**
    * The module name (if a module identity is used).
-   * If the module name contains any of the following characters,
-   * they must be percent-encoded as follows:
-   *         - `/` : `%2F`
-   *         - `%` : `%25`
-   *         - `#` : `%23`
-   *         - `&` : `%26`
+   * Must conform to the requirements of the MQTT spec for topic
+   * names (listed below) and of the IoT Hub (listed below)
+   * http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718106
+   * https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#device-identity-properties
```

Comment:
We should avoid including the locale and language in the URLs, so people get them in the regional settings on their browser/OS.

```suggestion
   * https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry#device-identity-properties
```

Reply:
ahhh yup this is usually the thing I point out in other people's pr's damn

> Why do you see the speck that is in your brother's eye, but do not notice the log that is in your own eye?

thanks!

Reply:
He who is without code errors, cast the first PR comment.